wouso/games/teamquest/tests.py

```diff
@@ -203,3 +205,39 @@ def test_quest_status_create(self):
 
     def test_quest_status_time_finished_before_time_started(self):
         pass
+
+
+class TeamQuestViewsTest(WousoTest):
+
+    def setUp(self):
+        self.admin = self._get_superuser()
+        self.c = Client()
+        self.c.login(username='admin', password='admin')
+
+    def test_cpanel_home_view_basic(self):
+        response = self.c.get(reverse('teamquest_home'))
+
+        self.assertEqual(response.status_code, 200)
+        self.assertContains(response, '<span class="glyphicon glyphicon-plus"></span>{% trans 'Add team quest' %}')
```

Comment:
It's better if you only test if the Name, Manage and the button exists (just the strings, no need to check for <th>, glyphicons, trans tags etc)